scripts/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+if [[ $PACKAGE == "distributive" ]] && ! which godep; then
+  go get -v github.com/tools/godep
+  go build -o tools/bin/godep github.com/tools/godep
```

Comment:
See line 313 of build: https://travis-ci.org/asteris-llc/mantl-packaging/jobs/113572361